### ISSUE TITLE
使用していないテーブルを洗い出す

### ISSUE THREAD

**Issue by haji33** [egg-system/olive-back#166]
## 概要
`coupon`テーブルのように使用していないテーブルが存在するので、それらを洗い出す

**Comment by haji33** [comment]
## 調査結果(2020/03/20)
### 判断基準
* 他のmodelの処理などで利用されていない
* 画面が存在しても導線が存在しない（またはコメントアウトなど）
* `/masters`以下の画面も使っている画面として判断

### 使用していないテーブル
* `coupon_histories`
  - 画面はあり( `/coupon_histories` )
  - 導線はコメントアウト( https://github.com/egg-system/olive-back/blob/master/src/app/views/masters/index.html.erb#L15 )
* `taxes`
  - 画面もなさそう

### 備考
* `coupon`は画面も導線も存在していました
  - 画面：`/coupons`
  - 導線：`/masters`のクーポンマスタ
  - reservation画面での利用箇所： https://github.com/egg-system/olive-back/blob/master/src/app/views/reservations/_form.html.erb#L18

**Comment by nuno929** [comment]
@haji33 
確認ですが、couponsは予約画面の入力欄と同じになっていますか？

**Comment by haji33** [comment]
@nuno929 
はい、reservationでは `reservation_coupons` 経由で同じ `coupons` を利用しています。

**Comment by nuno929** [comment]
@haji33 
こちらの件ですが、一度ER図を出力していただけないでしょうか？
そこから、使っていないテーブルを判断したく思います。
※ 外部キーだけ存在して、全く使っていないテーブルも多く存在するため

**Comment by haji33** [comment]
@nuno929 
承知しました。ER図はこちらになります。
※ `Rails ERD`を使って出力しました
[erd.pdf](https://github.com/egg-system/olive-back/files/4472375/erd.pdf)
